frontend/src/app/core/services/http-internal.service.ts

```diff
@@ -43,6 +43,13 @@ export class HttpInternalService {
         return this.httpClient.post<T>(this.buildUrl(url), payload, { headers: this.getHeaders() });
     }
 
+    public postFullRequest<T>(url: string, payload: object): Observable<HttpResponse<T>> {
```

Comment:
1. I don't like namings

Reply:
2. I don't think that we need observe part here, httpClient.post return observable 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/assets/57347711/7e4dfcda-97e8-49f0-a3a5-3b804e3bc9be)